spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -514,7 +515,7 @@ public void toHeaders(Headers source, Map<String, Object> target) {
 
 		Message<?> message = org.springframework.integration.support.MessageBuilder
 				.withPayload("foo")
-				.setHeader("foo", MimeTypeUtils.TEXT_PLAIN).build();
+				.setHeader("foo", MimeTypeUtils.TEXT_PLAIN.toString()).build();
```

Comment:
So, doesn't it work without a `.toString()`?

Reply:
It does. Let me see if i can revert.